src/Components/AddTodoForm.jsx

```diff
@@ -1,9 +1,21 @@
-export function AddTodoForm() {
+import React from "react";
+
+export function AddTodoForm(props) {
+    const [inputText, setInputText] = React.useState("");
+    let handleAddTodo = function (event) {
+        event.preventDefault();
+        let todoTitle = inputText;
```

Comment:
this works, but you could also just grab event.target.value instead of inputText